### ISSUE TITLE
Tower of Ascension Quest not giving the right amount of XP after completing

### ISSUE THREAD

**Issue by Timon96** [Wynncraft/Issues#6312]
I just finished the ToA quest on lvl 40 but only got to lvl 41. From the amount of XP you get on completing i should have been much higher after finishing it.
![2016-10-02_16 16 07](https://cloud.githubusercontent.com/assets/22575499/19021220/3f3233f0-88bd-11e6-9853-dde7a10ef7dc.png)
![2016-10-02_16 16 20](https://cloud.githubusercontent.com/assets/22575499/19021219/3f31c92e-88bd-11e6-83dc-49830149ecab.png)

The quest gives you 925 000 xp, i was at lvl 40 when i completed it. You need 46100 xp to get from 40 to 41. With 925k xp i should have gotten to lvl 51 with 30%. (You can see the values needed in the wiki https://wiki.wynncraft.com/index.php?title=Experience_Points). 

TLDR: I only got 46k xp from the 925k i should have gotten because of a bug.

**Comment by Moe-Ronickah** [comment]
From the Wiki: 375 000 xp.
[https://wiki.wynncraft.com/index.php?title=Tower_of_Ascension](https://wiki.wynncraft.com/index.php?title=Tower_of_Ascension)

ALSO, although the quest starts at Lv40, completing it at that level, impossible without obtaining fragments from another class, means that you lose 3%xp per level from 40 to where you should have completed it, at level 75.

**Comment by WhettoWen** [comment]
Does this apply to the hive aswell?

**Comment by Moe-Ronickah** [comment]
It applies to all quests, so if you try to circumvent the quest mechanics, and cheat your way to faster levels, you will lose xp.
